### PR TITLE
AG-12162 Add note annotation

### PR DESCRIPTION
**Pull request by lsjroberts**
https://ag-grid.atlassian.net/browse/AG-12162 

Also:

- Adds `_Scene.SvgPath` node e.g. `new SvgPath('M22 1.83333C22 0.820811...')`
- Fixes which color icons show for each annotation

### CODE REVIEW

File: packages/ag-charts-types/src/chart/toolbarOptions.ts [hunk 1]
```diff
@@ -60,7 +60,6 @@ export type AgIconName =
     | 'hollow-candlestick-series'
     | 'horizontal-line-drawing'
     | 'line-color'
-    | 'fill-color'
```
Comment (lsjroberts) [inline packages/ag-charts-types/src/chart/toolbarOptions.ts:63]: Duplicate of one above

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**